Makefile

```diff
@@ -21,7 +21,7 @@ BUILD_OUTPUT_DIRECTORY = $(BUILD_DIRECTORY)/out
 # ---------------------------------------------------------------------
 
 ELECTRON_VERSION = $(shell jq -r '.devDependencies["electron"]' package.json)
-NODE_VERSION = 6.1.0
+NODE_VERSION = $(shell node --version)
```

Comment:
The node version the builds target is the `ELECTRON_VERSION` (otherwise the native modules wouldn't work in Electron), unless the CLI is the target, in which case it's `NODE_VERSION`, and when the CLI is bundled and this version differs from the node that it's being bundled with then the native modules also won't work – hence this gets the actually installed node version now, to reduce places we have to manually change the node version.
I might just be totally wrong about this though – @jviotti will know for sure though.